### PR TITLE
[docs] Create a recipe about data-driven testing

### LINKED ISSUE #3268: Add example of the data-driven testing
Links:
* https://stackoverflow.com/questions/53953746/documentation-for-performing-data-driven-testing-using-testcafe
* https://testcafe-discuss.devexpress.com/t/multiple-execution-of-one-test-with-different-data/219/2

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
fixes #3268

### CODE REVIEW

**Comment by miherlosev** [comment]
ping @helen-dikareva

File: docs/articles/documentation/recipes/create-a-data-driven-test-case.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    }
+]
+```
+
+To create data-driven tests, iterate through the test cases, call the [test](../test-api/test-code-structure.md#tests) method at each iteration and write [test actions](../test-api/actions/README.md) and [assertions](../test-api/assertions/README.md) that use test case parameters.
+
+```js
+import { Selector } from 'testcafe';
+
+const dataSet = require('./data.json');
```
Comment (helen-dikareva) [inline docs/articles/documentation/recipes/create-a-data-driven-test-case.md]: we can use `import` here too

File: docs/articles/documentation/recipes/create-a-data-driven-test-case.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            .typeText('#developer-name', data.name)
+            .click('#tried-test-cafe')
+            .typeText('#comments', data.comment)
+            .click('#submit-button')
+            .expect(Selector('#article-header').textContent).eql(data.resultText);
+    });
+});
+```
+
+> **Tip:** you can store test cases in objects, arrays, [Maps](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map), [Sets](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set), etc., and iterate through them as you need. You can place them in a test file or import from `.json`, `.js`, `.ts`, etc.
```
Comment (helen-dikareva) [inline docs/articles/documentation/recipes/create-a-data-driven-test-case.md]: I'm not sure if it matters but we first told about `json` file in example and here we offer `Map` and `Set` . Would be it better to mention different file types at first and then different possibility to store data.

File: docs/articles/documentation/recipes/create-a-data-driven-test-case.md (hunk 1 above)
Comment (VasilyStrelyaev) [reply docs/articles/documentation/recipes/create-a-data-driven-test-case.md]: AFAIK, `import` doesn't parse JSON, does it?

File: docs/articles/documentation/recipes/create-a-data-driven-test-case.md (hunk 2 above)
Comment (VasilyStrelyaev) [reply docs/articles/documentation/recipes/create-a-data-driven-test-case.md]: Not sure either. Tips can be out of main context.

Comment (VasilyStrelyaev) [reply docs/articles/documentation/recipes/create-a-data-driven-test-case.md]: In this instance, it reads okay IMO

File: docs/articles/documentation/recipes/create-a-data-driven-test-case.md (hunk 1 above)
Comment (helen-dikareva) [reply docs/articles/documentation/recipes/create-a-data-driven-test-case.md]: you're right, but import also works fine in ES6. I don't mind leaving your variant.

**Pull request merged**